### PR TITLE
Update qualitatspass/compliance management page

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Follow up of #1581

This updates all of the Qualitatspass page to Compliance management.

### CODE REVIEW

File: source/localizable/kwaliteitspaspoort-new/_introduction.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<section class="capp-introduction" id="introduction">
+  <div class="row">
+    <div class="col">
+      <h3>
+        Ein anspruchsvolles Qualifikationsmanagement sollte die permanente Fort- und Weiterbildung Ihrer Mitarbeiter auch nachweisen können. Nur so können Sie Compliance sicherstellen.
```
Comment (sn3p) [inline source/localizable/kwaliteitspaspoort-new/_introduction.erb]: Is this file German only?

File: source/localizable/kwaliteitspaspoort-new/_overview.erb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<section class="capp-overview" id="features">
+
+  <div class="row">
+    <h2>
+      Warum CAPP Compliance & Qualifikationsmanagement?
```
Comment (sn3p) [inline source/localizable/kwaliteitspaspoort-new/_overview.erb]: idem?

Comment (RonaldDijkstra) [reply source/localizable/kwaliteitspaspoort-new/_overview.erb]: Yes

Comment (RonaldDijkstra) [reply source/localizable/kwaliteitspaspoort-new/_overview.erb]: screenshot 2019-01-28 at 17 20 34

Comment (RonaldDijkstra) [reply source/localizable/kwaliteitspaspoort-new/_overview.erb]: For now. Translations will come in when these pages are renewed in Dutch and English too.

Comment (RonaldDijkstra) [reply source/localizable/kwaliteitspaspoort-new/_overview.erb]: Ok done

**Pull request merged**